.github/workflows/forms-flow-web-cd.yml

```diff
@@ -1,92 +1,34 @@
-name: Push Forms flow Web to registry
-
+name: forms-flow-web-CD
 on:
   workflow_dispatch:
   push:
-    branches: [ master, develop, release/* ]
+    branches:
+      - master
```

Comment:
need to add develop also here. because later new code will merge to develop first. for testing and need to deploy this one as develop